rasa/core/training/visualization.py

```diff
@@ -261,7 +261,7 @@ def _merge_equivalent_nodes(graph, max_history):
     for s, e, k, d in edges:
         if k != EDGE_NONE_LABEL:
             if message_generator and d.get("label", k) is not None:
-                parsed_info = await interpreter.parse(d.get("label", k))
+                parsed_info = await interpreter.map_keyword_to_intent(d.get("label", k))
```

Comment:
I think this is different `parse`

Reply:
yes, sorry about that, I used pycharms refactor tool and it changed more than I wanted.
I already pushed a correction